ports/nrf/README.md

```diff
@@ -151,25 +151,25 @@ location.
-pca10056             | s140                    | Peripheral and Central |                | [Segger](#segger-targets)
-pca10059             | s140                    | Peripheral and Central | OpenBootloader | [nrfutil](#nrfutil-targets)
-particle_xenon       | s140                    | Peripheral and Central |                | [Black Magic Probe](#black-magic-probe-targets)
-nrf52840-mdk-usb-dongle | s140                 | Peripheral and Central | OpenBootloader | [nrfutil](#nrfutil-targets)
-pca10090             | None (bsdlib.a)         | None (LTE/GNSS)        |                | [Segger](#segger-targets)
-actinius_icarus      | None (bsdlib.a)         | None (LTE/GNSS)        |                | [Segger](#segger-targets)
+MICROBIT             | s110                    | Peripheral             |                | [PyOCD](#pyocdopenocd-targets)
+PCA10000             | s110                    | Peripheral             |                | [Segger](#segger-targets)
+PCA10001             | s110                    | Peripheral             |                | [Segger](#segger-targets)
+PCA10028             | s110                    | Peripheral             |                | [Segger](#segger-targets)
+PCA10031             | s110                    | Peripheral             |                | [Segger](#segger-targets)
+WT51822_S4AT         | s110                    | Peripheral             |                | Manual, see [datasheet](https://4tronix.co.uk/picobot2/WT51822-S4AT.pdf) for pinout
+PCA10040             | s132                    | Peripheral and Central |                | [Segger](#segger-targets)
+FEATHER52            | s132                    | Peripheral and Central |                | Manual, SWDIO and SWCLK solder points on the bottom side of the board
+ARDUINO_PRIMO        | s132                    | Peripheral and Central |                | [PyOCD](#pyocdopenocd-targets)
+IBK_BLYST_NANO       | s132                    | Peripheral and Central |                | [IDAP](#idap-midap-link-targets)
+IDK_BLYST_NANO       | s132                    | Peripheral and Central |                | [IDAP](#idap-midap-link-targets)
+BLUEIO_TAG_EVIM      | s132                    | Peripheral and Central |                | [IDAP](#idap-midap-link-targets)
+EVK_NINA_B1          | s132                    | Peripheral and Central |                | [Segger](#segger-targets)
+PCA10056             | s140                    | Peripheral and Central |                | [Segger](#segger-targets)
+PCA10059             | s140                    | Peripheral and Central | OpenBootloader | [nrfutil](#nrfutil-targets)
+PARTICLE_XENON       | s140                    | Peripheral and Central |                | [Black Magic Probe](#black-magic-probe-targets)
+NRF52840-MDK-USB-DONGLE | s140                 | Peripheral and Central | OpenBootloader | [nrfutil](#nrfutil-targets)
```

Comment:
should these hyphens be underscores?  (also below in this file)

Reply:
Fixed.